Plugin build silently exits when it can't find build config

Hi, I am the developer of [TubeArchivistMetadata](https://github.com/tubearchivist/tubearchivist-jf-plugin) and your tool doesn't work when I build my project, neither on local machine nor in GitHub Actions.
It returns me an empty string as result after the build and it doesn't really output any artifact.

Apart from this, I find this tool very poor documented unfortunately...

Nothing silent about this error?

Logs are sent to stderr, not stdout, and exit code is appropriately returning `1`, indicating a failure (`0` == ok, everything else indicates error).

```
$ jprm plugin build
Setting project version to 1.2.3.0
Setting project framework to net6.0
MSBuild version 17.4.8+6918b863a for .NET
MSBUILD : error MSB1003: Specify a project or solution file. The current working directory does not contain a project or solution file.
```

JPRM ran with debug logs shows the dotnet command;

```
$ jprm -v DEBUG plugin build 
debug: {'dotnet_config': 'Release', 'dotnet_framework': 'net6.0', 'output': '/tmp/tmp5ydb746i', 'max_cpu_count': 1, 'version': '1.2.3.0'}
Setting project version to 1.2.3.0
debug: Old version: 1.2.3.0
debug: Old file version: 1.2.3.0
debug: Old assembly version: 1.2.3.0
Setting project framework to net6.0
debug: ['run_os_command', ['dotnet', 'clean', '--configuration=Release', '--framework=net6.0'], None, False, '.']
MSBuild version 17.4.8+6918b863a for .NET
MSBUILD : error MSB1003: Specify a project or solution file. The current working directory does not contain a project or solution file.
```

JPRM is self-documenting to a certain level, via the `--help` argument, try appending it to different commands.
Otherwise, all of Jellyfin's official plugins serve as practical examples.

--- a/jprm/__init__.py
+++ b/jprm/__init__.py
@@ -100,6 +100,7 @@ def get_config(path):
             build_cfg = load_manifest(config_path)
             if build_cfg is not None:
                 return build_cfg
+    logger.warning("Failed to locate config file.")
     return None
 
 
@@ -820,10 +821,14 @@ def cli_plugin():
     help='Max number of cores to use during build (1)',
 )
 def cli_plugin_build(path, output, dotnet_configuration, dotnet_framework, max_cpu_count, version):
+    build_cfg = get_config(path)
+    if build_cfg is None:
+        raise click.UsageError('No build config found in `{}`'.format(path))
+
     with tempfile.TemporaryDirectory() as bintemp:
-        build_plugin(path, output=bintemp, dotnet_config=dotnet_configuration, dotnet_framework=dotnet_framework,
+        build_plugin(path, output=bintemp, build_cfg=build_cfg, dotnet_config=dotnet_configuration, dotnet_framework=dotnet_framework,
                      version=version, max_cpu_count=max_cpu_count)
-        filename = package_plugin(path, version=version, binary_path=bintemp, output=output)
+        filename = package_plugin(path, build_cfg=build_cfg, version=version, binary_path=bintemp, output=output)
         click.echo(filename)
 
 

